src/components/transactions/TxDetails/index.tsx

```diff
@@ -109,7 +109,7 @@ const TxDetailsBlock = ({ txSummary, txDetails }: TxDetailsProps): ReactElement
           <TxSigners txDetails={txDetails} txSummary={txSummary} />
 
           {isQueue && (
-            <Box display="flex" alignItems="center" justifyContent="center" gap={1} mt={2}>
+            <Box className={`${css.buttons}`}>
```

Comment:
```suggestion
            <Box className={css.buttons}>
```

Reply:
Right! I have modified the unnecesary template strings found in the file.